### PR TITLE
Drawable updates

### PR DESCRIPTION
**Pull request by TacoTheDank**
- Regenerated some drawables to their latest material design
- Deleted some duplicate drawables where it was possible

### CODE REVIEW

**Comment by cooltey** [comment]
Please resolve the conflicts. Thank you.

**Comment by TacoTheDank** [comment]
> Please resolve the conflicts. Thank you.

Done :)

File: app/src/main/res/drawable/ic_zoom_in_themed_24dp.xml [hunk 1]
```diff
@@ -1,5 +1,13 @@
...
-    android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
-    <path android:fillColor="?attr/toolbar_icon_color" android:pathData="M15.5,14h-0.79l-0.28,-0.27C15.41,12.59 16,11.11 16,9.5 16,5.91 13.09,3 9.5,3S3,5.91 3,9.5 5.91,16 9.5,16c1.61,0 3.09,-0.59 4.23,-1.57l0.27,0.28v0.79l5,4.99L20.49,19l-4.99,-5zM9.5,14C7.01,14 5,11.99 5,9.5S7.01,5 9.5,5 14,7.01 14,9.5 11.99,14 9.5,14zM12,10h-2v2L9,12v-2L7,10L7,9h2L9,7h1v2h2v1z"/>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:autoMirrored="true"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+    <path
+        android:fillColor="@android:color/white"
```
Comment (cooltey) [inline app/src/main/res/drawable/ic_zoom_in_themed_24dp.xml:8]: The color here should be `?attr/toolbar_icon_color`.

File: app/src/main/res/drawable/ic_zoom_in_themed_24dp.xml [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+        android:pathData="M15.5,14h-0.79l-0.28,-0.27C15.41,12.59 16,11.11 16,9.5 16,5.91 13.09,3 9.5,3S3,5.91 3,9.5 5.91,16 9.5,16c1.61,0 3.09,-0.59 4.23,-1.57l0.27,0.28v0.79l5,4.99L20.49,19l-4.99,-5zM9.5,14C7.01,14 5,11.99 5,9.5S7.01,5 9.5,5 14,7.01 14,9.5 11.99,14 9.5,14z" />
+    <path
+        android:fillColor="@android:color/white"
```
Comment (cooltey) [inline app/src/main/res/drawable/ic_zoom_in_themed_24dp.xml:11]: The color here should be `?attr/toolbar_icon_color`.

**Review by cooltey — CHANGES_REQUESTED** [review]

File: app/src/main/res/drawable/ic_zoom_in_themed_24dp.xml (hunk 2 above)
Comment (TacoTheDank) [reply app/src/main/res/drawable/ic_zoom_in_themed_24dp.xml]: Fixed.

**Pull request merged**